### ISSUE TITLE
Training: Small scale experiments in Oslofjorden

### ISSUE THREAD

**Issue by mateuszmatu** [metno/ocean-ai#290]
I created a graph for Oslofjorden. I expect runs in this area to be worse than full-scale runs, but it's much faster so we should use this to test implementations and see what sticks. Some ideas I had in mind:

**Graphs**
- [x] Controll run res 12 graph
- [x] Res 10 graph
- [ ] hexagonal
- [ ] no land
- [x] res 11

**Variables**
- [x] Add zeta to training again. 
- [ ] salt anomalies
- [ ] salt --> density

**Rollout**
- [ ] Iterative increase of rollout length
- [ ] 100 000 step rollout training with eg. rollout=3
- [ ] Change learning rate

Find a setup that seems to be good and do a very long training with it, e.g. 500 000 steps. Then perform overfitting analysis #149

**Comment by mateuszmatu** [comment]
Ran a few of these. it's good to check whether the setup runs, but the results in the domain are bad.
